gbpservice/nfp/orchestrator/modules/service_orchestrator.py

```diff
@@ -224,7 +227,8 @@ def __init__(self, conf, controller):
                      'POLICY_TARGET_ADD',
                      'POLICY_TARGET_REMOVE',
                      'CONSUMER_ADD',
-                     'CONSUMER_REMOVE']
+                     'CONSUMER_REMOVE',
+                     'UPDATE_USER_CONFIG']
```

Comment:
It will be clear if the events are ordered according to the names. I mean UPDATE_USER_CONFIG can be after DELETE_USER_CONFIG in the list.

But it involves changes in other event paths. We can check what amount of changes involves and if they are less, better to do the changes accordingly.